build.sbt

```diff
@@ -14,41 +14,39 @@
  * limitations under the License.
  */
 
-import Dependencies._
-import JacocoSetup._
 import sbt.Keys.name
+import sbt.*
+import Dependencies.*
+import Dependencies.Versions.spark3
+import VersionAxes.*
 
 ThisBuild / organization := "za.co.absa.atum-service"
 sonatypeProfileName := "za.co.absa"
 
-ThisBuild / scalaVersion := Versions.scala213  // default version
+ThisBuild / scalaVersion := Setup.scala213  // default version
 
 ThisBuild / versionScheme := Some("early-semver")
 
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
 publish / skip := true
 
-lazy val printSparkScalaVersion = taskKey[Unit]("Print Spark and Scala versions for atum-service is being built for.")
-lazy val printScalaVersion = taskKey[Unit]("Print Scala versions for atum-service is being built for.")
+initialize := {
+  val _ = initialize.value // Ensure previous initializations are run
+
+  val requiredJavaVersion = VersionNumber("17")
+  val current = VersionNumber(sys.props("java.specification.version"))
```

Comment:
Consider renaming to something like `javaVersion` or `javaVersionInUse`.

Reply:
If code kept, will rename.